device/mellanox/x86_64-nvidia_sn2201-r0/platform.json

```diff
@@ -0,0 +1,483 @@
+            "lanes": "132",
+            "breakout_modes": {
+                "1x1G[100]": ["etp34"]
+            }
+        },
+        "Ethernet34": {
+            "index": "35",
+            "lanes": "136",
+            "breakout_modes": {
+                "1x1G[100]": ["etp35"]
+            }
+        },
+        "Ethernet35": {
+            "index": "36",
+            "lanes": "140",
+            "breakout_modes": {
+                "1x1G[100]": ["etp36"]
+            }
+        },
+        "Ethernet36": {
+            "index": "37",
+            "lanes": "144",
+            "breakout_modes": {
+                "1x1G[100]": ["etp37"]
+            }
+        },
+        "Ethernet37": {
+            "index": "38",
+            "lanes": "148",
+            "breakout_modes": {
+                "1x1G[100]": ["etp38"]
+            }
+        },
+        "Ethernet38": {
+            "index": "39",
+            "lanes": "152",
+            "breakout_modes": {
+                "1x1G[100]": ["etp39"]
+            }
+        },
+        "Ethernet39": {
+            "index": "40",
+            "lanes": "156",
+            "breakout_modes": {
+                "1x1G[100]": ["etp40"]
+            }
+        },
+        "Ethernet40": {
+            "index": "41",
+            "lanes": "160",
+            "breakout_modes": {
+                "1x1G[100]": ["etp41"]
+            }
+        },
+        "Ethernet41": {
+            "index": "42",
+            "lanes": "164",
+            "breakout_modes": {
+                "1x1G[100]": ["etp42"]
+            }
+        },
+        "Ethernet42": {
+            "index": "43",
+            "lanes": "168",
+            "breakout_modes": {
+                "1x1G[100]": ["etp43"]
+            }
+        },
+        "Ethernet43": {
+            "index": "44",
+            "lanes": "172",
+            "breakout_modes": {
+                "1x1G[100]": ["etp44"]
+            }
+        },
+        "Ethernet44": {
+            "index": "45",
+            "lanes": "176",
+            "breakout_modes": {
+                "1x1G[100]": ["etp45"]
+            }
+        },
+        "Ethernet45": {
+            "index": "46",
+            "lanes": "180",
+            "breakout_modes": {
+                "1x1G[100]": ["etp46"]
+            }
+        },
+        "Ethernet46": {
+            "index": "47",
+            "lanes": "184",
+            "breakout_modes": {
+                "1x1G[100]": ["etp47"]
+            }
+        },
+        "Ethernet47": {
+            "index": "48",
+            "lanes": "188",
+            "breakout_modes": {
+                "1x1G[100]": ["etp48"]
+            }
+        },
+        "Ethernet48": {
+            "index": "49,49,49,49",
+            "lanes": "192,193,194,195",
+            "breakout_modes": {
+                "1x100G[50G,40G,25G,10G]": ["etp49"],
+                "2x50G[25G,10G]": ["etp49a", "etp49b"],
+                "4x25G[10G]": ["etp49a", "etp49b", "etp49c", "etp49d"]
```

Comment:
for now this cannot be supported as when you do so, the other port must removed and there is no ability to support it on the dynamic port breakout. so as other systems of spectrum we don't provide the ability to do so.
Check SN2700 for example. Same behaviour.

Reply:
Fixed.